src/etc/inc/plugins.inc.d/ntpd.inc

```diff
@@ -396,7 +396,7 @@ function ntpd_configure_do($verbose = false)
     if (empty($config['ntpd']['nomodify'])) { /*note: this one works backwards */
         $ntpaccess .= ' nomodify';
     }
-    if (!empty($config['ntpd']['noquery'])) {
+    if (empty($config['ntpd']['noquery'])) { /*note: this one works backwards */
```

Comment:
On a vanilla 24.7.3 the checkbox on the GUI does what I expect it to do WRT ntpd.conf but something here is definitely not right.

Reply:
Yep, see my additional comment. It works properly now and to remain properly working after switching the defaults, users who enabled it need to have the configuration unset in config.xml and all of this stuff needs to go backwards to remain consistently inconsistent with the rest. 🤣Those who did not enable it should be ok without doing anything and without any need to do any migration for them and with sane new default. . If they want to re-enable the query "feature", then they need to change (untick) the checkbox after upgrade themselves (release notes item probably).

Reply:
perhaps the best course of action is park this on devel for eventual 25.1 inclusion with a "migration note" in the release notes that the old "noquery" (opt-in) is no longer in effect and we switched to a new default (opt-out)